test/model/test_forecast.py

```diff
@@ -17,7 +17,7 @@
 import pytest
 
 # First-party imports
-from gluonts.model.forecast import QuantileForecast, SampleForecast
+from gluonts.model.forecast import Forecast, QuantileForecast, SampleForecast
```

Comment:
```suggestion
from gluonts.model.forecast import QuantileForecast, SampleForecast
```

This is not needed anymore.

Reply:
Yep. Removed.